pyccel/ast/core.py

```diff
@@ -4228,3 +4228,4 @@ def doit(expr, targets):
 
 #==============================================================================
 
+
```

Comment:
Useless whitespace change

Reply:
Thank you! I will fix it right away

Reply:
My point was that this file should not be modified at all by this PR. I've fixed it.